pkg/workers/share/share.go

```diff
@@ -23,20 +32,42 @@ func init() {
 		// retry, but with MaxExecCount > 1, it can amplifies a lot the number
 		// of retries
 		MaxExecCount: 1,
-		Timeout:      300 * time.Second,
+		Timeout:      5 * time.Minute,
 		WorkerFunc:   WorkerReplicate,
 	})
 
 	// TODO write documentation about this worker
 	jobs.AddWorker(&jobs.WorkerConfig{
-		WorkerType:   "share-track",
-		Concurrency:  runtime.NumCPU(),
-		MaxExecCount: 2,
-		Timeout:      30 * time.Second,
-		WorkerFunc:   WorkerTrack,
+		WorkerType:  "share-upload",
+		Concurrency: runtime.NumCPU(),
+		// TODO the worker is not idempotent: if it fails, it adds a new job to
+		// retry, but with MaxExecCount > 1, it can amplifies a lot the number
+		// of retries
+		MaxExecCount: 1,
+		Timeout:      1 * time.Hour,
```

Comment:
A large video might fail with a slow connection, but do we support this kind of case ?

Reply:
We will support this kind of case, but latter. For the moment, the limit of 1 hour seems to be a good tradeoff.